### PR TITLE
Add an user facts module for Online

### PR DESCRIPTION
**Pull request by remyleone**
##### SUMMARY

Add an user facts module for Online

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

- online
- online_user_facts

##### ANSIBLE VERSION

```
$ ansible --version
ansible 2.7.0.dev0 (online_user a88758d7e7) last updated 2018/08/27 10:55:11 (GMT +200)
  config file = None
  configured module search path = [u'/Users/sieben/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/sieben/workspace/ansible/lib/ansible
  executable location = /Users/sieben/workspace/ansible/bin/ansible
  python version = 2.7.15 (default, Jun 17 2018, 12:46:58) [GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)]
```

### CODE REVIEW

**Review by Spredzy — APPROVED** [review]
LGTM.

shipit

**Comment by remyleone** [comment]
shipit

**Comment by maxamillion** [comment]
rebuild_merge

File: lib/ansible/module_utils/online.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import json
```
Comment (abadger) [inline lib/ansible/module_utils/online.py:1]: Need to put moudle_utils files under a BSD license.  (See lib/ansible/module_utils/basic.py for an example of the header we use).

Also need boilerplate:
```

from __future__ import (absolute_import, division, print_function)
__metaclass__ = type
```

File: lib/ansible/module_utils/online.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    )
+
+
+class OnlineException(Exception):
+
+    def __init__(self, message):
+        self.message = message
+
+
+class Response(object):
```
Comment (abadger) [inline lib/ansible/module_utils/online.py:24]: You don't have to inherit from object if you have the boilerplate ```__metaclass__ = type```

File: lib/ansible/module_utils/online.py (hunk 1 above)
Comment (remyleone) [reply lib/ansible/module_utils/online.py:1]: I've opened a PR to fix it https://github.com/ansible/ansible/pull/45106

**Pull request merged**